### DISCUSSION TITLE
Templating in panels does not work when creating alerts using suggested values

### DISCUSSION THREAD

**Discussion by mbrancato** [grafana/grafana#63329] [Alerting]
When a panel as a templated variable in the title, this variable is not templated in the suggested "rule name" for alerting. It subsequently is not evaluated when an alert notification is sent.

Example panel:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/8493275/218616063-bea3d932-fae7-46a4-aac6-40de3f430f80.png">

Which has the panel title defined as:
```Disk writes/reads ($instance)```

When editing the panel, using the button `Create alert rule from this panel` results in a suggested rule title:
```Disk writes/reads ($instance)```

This is deceiving as the variable is not templated in the rule alert notifications. I think there should be a better flow here where an actual templated value is passed in the notification

Either way, if the capabilities of the panel title do not translate to the alert rule notification title, then some sort of translation or conversion should happen when crating these via the UI.

**Comment by gillesdemey** [comment]
This sounds like an actual bug and I can reproduce this one, I've created an issue for it here 👉 https://github.com/grafana/grafana/issues/63796
